README.md

```diff
@@ -1,6 +1,21 @@
-# web3-ui
-This will be a library made up of UI components specifically crafted for web3 use cases.
+# Welcome to web3-ui (The UI library for Web3)
+
+web3-ui is a library made up of UI components specifically crafted for web3 use cases.
 
 [Read the RFC](https://forum.developerdao.com/t/rfc-web3-ui-a-web3-specific-ui-library)
 
+## Installation
+
+web3-ui is available as an [npm package](https://www.npmjs.com/package/web3-ui)
+
+Install using:
+
+```bash
+yarn add web3-ui;
+or
+npm i web3-ui;
+```
```

Comment:
@Dhaiwat10 can you verify if these are correct, or did we move the packages to the @web3-ui namespace

Reply:
We haven't actually published the components package yet. Will do that once some components are ready and ping here.